src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -299,9 +299,27 @@
 
 (rf/reg-event-fx :wallet/disable-from-networks
  (fn [{:keys [db]} [chain-ids]]
-   {:db (-> db
-            (assoc-in [:wallet :ui :send :disabled-from-chain-ids] chain-ids)
-            (assoc-in [:wallet :ui :send :loading-suggested-routes?] true))}))
+   {:db (assoc-in db [:wallet :ui :send :disabled-from-chain-ids] chain-ids)}))
+
+(rf/reg-event-fx :wallet/reset-network-amounts-to-zero
```

Comment:
is this the value set per network? 🤔 

Probably @vkjr will need a similar function to set specific network values in: https://github.com/status-im/status-mobile/issues/16974


Maybe we can align the two methods/approaches if needs be?

Reply:
I think for #16974 we would just need to set a locked-network-values map and maybe modify network-amounts utils function to handle a locked type to render in the card. This utils is very simple, just map and modify current values to zero.